app.py

```diff
@@ -0,0 +1,23 @@
+from flask import Flask, jsonify, request#, url_for
```

Comment:
What's the purpose of this in a Nuxt app?

Reply:
Just setting up a backend in case it's needed later.